### ISSUE TITLE
Oracle Result Setting at this place should be a cross instead of tick

### ISSUE THREAD

**Issue by henryzbw** [bodhiproject/bodhi-ui#113]
![image](https://user-images.githubusercontent.com/10339131/35546690-1ec656c4-052a-11e8-9360-4d62f854dc92.png)

**Comment by liviavinci** [comment]
What does this issue mean @dwalintukan

**Comment by dwalintukan** [comment]
@liviavinci I think he was suggesting to use an X where the bubbles are for stages that are complete. I personally like the checkmark.

**Comment by henryzbw** [comment]
@liviavinci I was suggesting to use an X for the Oracle Result Setting in this page since the result setter doesn't set the result

**Comment by dwalintukan** [comment]
@henryzbw do you mean when a user is not the result setter and they are in the `Set Result` page?

**Comment by henryzbw** [comment]
@dwalintukan No, if we enter the open result set phase, whether a user is the result setter or not, the oracle result setting should always be a cross. the cross only means the result setter doesn't set the result. If the result setter sets the result, we won't go to open result set phase, oracle result setting should be a tick.

**Comment by dwalintukan** [comment]
Ah I see what you mean. That makes sense to me.
